SubcategoriesForCAP/examples/PrecompileSliceCategory.g

```diff
@@ -19,6 +19,22 @@ CapJitAddLogicTemplate(
     )
 );
 
+CapJitAddLogicTemplate(
+    rec(
+        variable_names := [ "matrix", "dimension", "ring" ],
+        src_template := "matrix * HomalgIdentityMatrix( dimension, ring )",
+        dst_template := "matrix",
+    )
+);
+
+CapJitAddLogicTemplate(
+    rec(
+        variable_names := [ "matrix", "dimension", "ring" ],
+        src_template := "HomalgIdentityMatrix( dimension, ring ) * matrix",
+        dst_template := "matrix",
+    )
+);
+
```

Comment:
The following would make the diff smaller:
```suggestion

# Length( [ 1 .. n ] ) -> n
CapJitAddLogicTemplate(
    rec(
        variable_names := [ "n" ],
        src_template := "Length( [ 1 .. n ] )",
        dst_template := "n"
    )
);
```

Reply:
Thank you, Fabian.